tox.ini

```diff
@@ -92,7 +92,7 @@ envlist =
     ; below mean these dependencies are being used:
     ; 0: elasticsearch-dsl==6.4.0 elasticsearch==6.8.2
     ; 1: elasticsearch-dsl==7.4.1 elasticsearch==7.17.9
-    ; 2: elasticsearch-dsl>=8.0,<8.13 elasticsearch>=8.0,<8.13
+    ; 2: elasticsearch-dsl==8.13.1 elasticsearch==8.13.1
```

Comment:
Are we still testing versions 8.0 to 8.12?

Reply:
Nope, i tested it manually. Do you think it would be different enough we need to run separate tests? Thinking about it the test should fail with elasticsearch < 8.13

Reply:
As long as we're testing a version 8 client, it's fine. I was worried that by only using 8.13 we would not exercise the code path used in 8.12, for example.

Reply:
We are running all tests with native otel disabled and just one with it enabled